docs/rbd-nbd.md

```diff
@@ -0,0 +1,48 @@
+# RBD NBD Mounter
+
+- [RBD NBD Mounter](#rbd-nbd-mounter)
+  - [Overview](#overview)
+  - [Configuration](#configuration)
+  - [Status](#status)
+  - [Support Matrix](#support-matrix)
+    - [CSI spec and Kubernetes version compatibility](#csi-spec-and-kubernetes-version-compatibility)
+
+## Overview
+
+The RBD CSI plugin will provision new RBD images and attach and mount those
+to workloads. Currently, the default mounter is krbd, which uses the kernel
+rbd driver to mount the rbd images onto the application node. Here on
+at ceph-csi we will also have a userspace way of mounting the RBD images,
+via RBD-NBD.
+
+[Rbd-nbd](https://docs.ceph.com/en/latest/man/8/rbd-nbd/) is a client for
+RADOS block device (rbd) images like the existing rbd kernel module. It
+will map an rbd image to an NBD (Network Block Device) device, allowing
+access to it as a regular local block device.
+
+It’s worth to make a note that the rbd-nbd processes will run on the
+client-side, which is inside the `csi-rbdplugin` node plugin.
+
+## Configuration
+
+To use the rbd-nbd mounter for RBD-backed PVs, set `mounter` to `rbd-nbd`
+in the StorageClass.
+
+## Status
+
+Rbd-nbd support status: **Alpha**
```

Comment:
this should probably be linked from the main [https://github.com/ceph/ceph-csi/blob/devel/README.md#support-matrix](support matrix).

Reply:
Good idea, I will refresh.

Reply:
Need to mention other operations? like CreateVolume ,expand etc?

Reply:
We are not done on this Madhu, as you there are issues opened for qualifying encryption, expansion, thick provisioning, mirroring for rbd-nbd. With each of these issues, we are going to work closely with this doc again.

But, if you want Create Volume, Delete Volume to be added with this PR, yes that also make sense too, but just saying we are not done with this doc yet. I will leave it to you, please feel free to let me know your further thought on this.

Thanks!

Reply:
Am fine if we are adding it later also

Reply:
Cool. Sure @Madhu-1, we also have one issue for verifying all access modes and volume modes for rbd-nbd, may be we can test on access modes and volumes modes and then add support matrix accordingly for volume provisioning too.